### DISCUSSION TITLE
Make the app Web based

### DISCUSSION THREAD

**Discussion by MananGandhi1810** [ThioJoe/YT-Spammer-Purge#660] [Ideas]
Hello. I was reading the comments on the first video in which Joe discusses about the app, when i stumbled upon this comment 
![image](https://user-images.githubusercontent.com/67302570/154103648-3f0859ed-afc0-4ef1-b5bd-8b99f3c73769.png)

I feel this is a good suggestion and could be less hassle to set up for the users rather than the desktop app method.

Credit for suggestion - Arjun Dureja (https://www.youtube.com/channel/UC1oNaT4soSIf0-TmhxqqF-w)

**Comment by ThioJoe** [comment]
That would be a lot of work

**Reply by MananGandhi1810** [reply]
I can help you make the frontend, if you can convert the app into some kind of an API. I am not very familiar with the youtube API

**Reply by somecoderithink** [reply]
> 

Well, I can also help making it into a web app (coding of websites is my hobby).

**Comment by Firecul** [comment]
There has been some discussion on this in #84 also
